### DISCUSSION TITLE
Casdoor的admin默认密码不是123？

### DISCUSSION THREAD

**Discussion by wuhuanyan** [lobehub/lobehub#4890] [General Question] [answered]
https://lobehub.com/zh/docs/self-hosting/server-database/docker-compose
我少做了哪一步？注册又显示用户已存在。
![image](https://github.com/user-attachments/assets/10eed172-fdaa-4f25-be0a-ed64a3e331c5)

**Comment by SpeedupMaster** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/460bf79c-9b96-45d3-a051-256f6309c28a)
更新后好像会重新生成密码

**Reply by wokaomai** [reply]
这段代码在什么位置

**Reply by SpeedupMaster** [reply]
> 这段代码在什么位置

脚本里

**Comment by wuhuanyan** [comment]
好的，我找到了
